src/components/CommentList/CommentList.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
 import './CommentList.scss';
+import { Comment } from '../../react-app-env';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="comments">
+    {comments.map(com => <CommentInfo comment={com} />)}
```

Comment:
add key to CommentInfo

https://reactjs.org/docs/lists-and-keys.html
```suggestion
    {comments.map(comment => <CommentInfo comment={comment} />)}
```